src/sentry/api/endpoints/project_rule_enable.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import audit_log
+from sentry.api.api_owners import ApiOwner
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectAlertRulePermission, ProjectEndpoint
+from sentry.api.endpoints.project_rules import find_duplicate_rule
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.constants import ObjectStatus
+from sentry.models import Rule
+
+
+@region_silo_endpoint
+class ProjectRuleEnableEndpoint(ProjectEndpoint):
+    owner = ApiOwner.ISSUES
+    permission_classes = (ProjectAlertRulePermission,)
+
+    def put(self, request: Request, project, rule_id) -> Response:
+        try:
+            rule = Rule.objects.get(id=rule_id, project=project)
+        except Rule.DoesNotExist:
+            raise ResourceDoesNotExist
+
+        if rule.status != ObjectStatus.DISABLED:
+            return Response(
+                {
+                    "detail": "Rule is not disabled.",
+                },
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        if not rule.data.get("actions", []):
+            return Response(
+                {
+                    "detail": "Cannot enable a rule with no action.",
+                },
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        kwargs = {**rule.data}
```

Comment:
why do we have to make a copy here?

Reply:
I could rewrite it but I can't pass it like `find_duplicate_rule(**rule.data, project)` - I initially wrote this function to be used by the other project rule endpoints.

Reply:
why can't you just pass it like `find_duplicate_rule(**rule.data, project)`?

Reply:
`positional argument follows keyword argument unpacking` 😛 I can flip the args if you want though

Reply:
whoops, sorry mean this: `find_duplicate_rule(rule.data, project)`